pyproject.toml

```diff
@@ -52,14 +43,21 @@ ipfabric = { version = "~6.0.9", optional = true }
 ipfabric-diagrams = { version = "~6.0.2", optional = true }
 isodate =  { version = "^0.6.1", optional = true }
 meraki = { version = "^1.7.2", optional = true }
+nautobot = "^1.4.0"
+nautobot-capacity-metrics = "*"
 netmiko = { version = "^3.4.0", optional = true }
 netutils = { version = "^1.1.0", optional = true }
 pan-os-python = { version = "^1.3.0", optional = true }
 prettytable = { version = "^2.1.0", optional = true }
 protobuf = { version = "^3.17", optional = true }
 pydantic = { version = "^1.8.2", optional = true }
+python = "^3.7.1"
```

Comment:
If we're doing non zero minimum versions, might as well consider bumping to 3.7.2 so the newer pylint gets used.